src/chef-mover/src/mover.app.src

```diff
@@ -7,10 +7,19 @@
                   stdlib,
                   sasl,
                   crypto,
+                  envy,
                   inets,
                   ibrowse,
                   lager,
                   bcrypt,
+                  moser,
+                  depsolver,
```

Comment:
A lot of these are actually transitive dependencies of moser.  We should probably declare them there.